core/sqf/sqenvcom.sh

```diff
@@ -580,6 +547,13 @@ EOF
     echo "**** ERROR: Unable to determine location of HBase lib directory"
   fi
 
+  if [[ -d $TOOLSDIR/thrift-0.9.0 ]]; then
+    # this is mostly for a build environment, where we need
+    # thrift from TOOLSDIR
+    export THRIFT_LIB_DIR=$TOOLSDIR/thrift-0.9.0/lib
+    export THRIFT_INC_DIR=$TOOLSDIR/thrift-0.9.0/include
+  fi
+
   if [ -n "$HBASE_CNF_DIR" -a -n "$HADOOP_CNF_DIR" -a \
```

Comment:
Is CURL_INC_DIR and CURL_LIB_DIR removed ? We need it for LOB support. The code is not yet active due to a few issues hence no regression test yet for this case.